src/View/Helper/FormHelper.php

```diff
@@ -2480,18 +2487,29 @@ public function getValueSources(): array
         return $this->_valueSources;
     }
 
+    /**
+     * Extracts valid value sources.
+     *
+     * @param string|string[] $sources A string or a list of strings identifying a source.
+     * @return array
```

Comment:
string[] then?

Reply:
It receives the same type of setValueSources()

Reply:
I don't think the return result does, if you look into what the code does.